jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -98,8 +103,11 @@ class CawsConnectionProvider : GatewayConnectionProvider {
             return null
         }
 
+        val ssoSettings = connectionParams.ssoSettings ?: SsoSettings(SONO_URL, SONO_REGION)
```

Comment:
if we're defaulting to builder id, then https://github.com/aws/aws-toolkit-jetbrains/blob/4f5f56d08077653ce216b829d5354a86ad12d1f7/jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/welcomescreen/WorkspaceDetails.kt#L140-L149 needs to be amended to include the sso url / region when connecting from the gateway UI